test/pushnotificationstestutils.cpp

```diff
@@ -64,7 +98,34 @@ void FakeWebSocketServer::socketDisconnected()
     }
 }
 
-OCC::AccountPtr FakeWebSocketServer::createAccount()
+void FakeWebSocketServer::waitForTextMessages() const
+{
+    QVERIFY(_processTextMessageSpy->wait());
```

Comment:
So the problem of using QVERIFY like that, is that it will always point to this line for the failing tests, this won't be very useful since you won't necessarily know where in the test you are (if the test calls this several times). So better return a bool here and use QVERIFY in the caller.

This also allows the test to not necessarily fail on timeouts, sometimes that's even what you want.